cg/cli/demultiplex/sample_sheet.py

```diff
@@ -29,19 +25,28 @@ def sample_sheet_commands():
 
 
 @sample_sheet_commands.command(name="validate")
+@click.argument("flow-cell-name")
 @click.argument("sheet", type=click.Path(exists=True, dir_okay=False))
 @OPTION_BCL_CONVERTER
+@click.pass_obj
 def validate_sample_sheet(
+    context: CGConfig,
+    flow_cell_name: str,
     bcl_converter: str,
     sheet: click.Path,
 ):
-    """Validate a sample sheet."""
+    """Validate a sample sheet.
+    flow-cell-name is the flow cell run directory name, e.g. '201203_A00689_0200_AHVKJCDRXX'
```

Comment:
Maybe add this as a description of the option when using --help as well?

Reply:
yes, this is actually printed when you call `--help`, see the beginning of the testing on stage in the comment above